README.md

```diff
@@ -148,25 +148,30 @@ You can then apply the enforced style by typing:
 make format
 ```
 
-# Unit tests
+# Unit Tests
 
-Some initial effort is underway to start providing unit tests for our tooling, starting with mega65_ftp.
+Some initial effort is underway to start providing unit tests for our tooling, starting with mega65_ftp and bit2core.
 
 It presently makes use of gtest release 1.10.0.
 
+To install gtest on linux run the following:
+```
+sudo apt install cmake
+git clone https://github.com/google/googletest.git -b release-1.10.0
+cd googletest
+mkdir build
+cd build
+cmake ..
+make
+sudo make install
+```
+
 Unit tests are housed in the 'gtest/' folder.
 
 The unit test executables are housed in the 'gtest/bin' folder.
 
-E.g. to generate the executable:
-
-```
-make gtest/bin/mega65_ftp.test
-```
-
-To run the unit tests:
+To generate the tests and run them:
 
 ```
-cd gtest/bin
-./mega65_ftp.test
+make test
```

Comment:
Maybe mention both linux and windows methods:

For linux: `make test`
For windows: `make test.exe`

Reply:
Updated.